steward_proto/proto/aave_v2_stablecoin.proto

```diff
@@ -48,11 +50,24 @@ message AaveV2Stablecoin {
     message ClaimAndUnstake {}
 
     /*
-     * Enters inactive assets into the current Aave stablecoin position.
+     * Pushes assets into the current Aave lending position.
      *
-     * Represents function `enterPosition()`
+     * Represents function `enterPosition(uint256 assets)`
      */
-    message EnterPosition {}
+    message EnterPosition {
```

Comment:
The mock contract doesn't include arguments for `EnterPosition` or `ExitPosition`. Should they be there? The Steward rust code and generated docs also don't seem to indicate the presence of arguments.

Reply:
So turns out the functions `enterPositions` and `exitPosition` are overloaded in the contract and the abi build used the wrong signature.